lib/DirController.ts

```diff
@@ -7,22 +7,31 @@ import type { ReactiveController, ReactiveControllerHost } from 'lit';
  */
 export class DirController implements ReactiveController {
   /** The element's current `dir` */
-  public dir: string;
+  public dir: 'ltr'|'rtl'|'auto' = 'auto';
```

Comment:
Please lint.

If you use VSCode, please enable eslint fix-all-on-save with this in your settings.json

```json
    "editor.codeActionsOnSave": {
        "source.fixAll.eslint": true
    }
```